File: Python/Input/prg19.py
#you must enter exactly two values because there are two variables to take input

roll,aadhar = [int(x) for x in input('Please enter aadhar and roll no\n').split()]
print(roll,type(roll))
print(aadhar,type(aadhar))

print('the roll is ',roll)
print('the aadhar is ',aadhar)


print('the roll is {} the aadhar is {}'.format(roll,aadhar))

File: C/Statements/Switch/prg19.c
#include<stdio.h>

void main() {

    int a = 10;

    switch(a) {

        case 9 : {
            int x = 20;
            int y = 10;
            printf("%d\n", x + y);
        }

        // " {} " are must here for cases
        case 10 : {
            int x = 20;
            int y = 10;
            printf("%d\n", x - y);
        }
    }
}
